.travis.yml

```diff
@@ -12,6 +12,7 @@ jobs:
         - black --check . || true 
         - flake8 --ignore=E203,W503 --max-complexity=25 --max-line-length=88 --statistics --count .
         - scripts/validate_filenames.py  # no uppercase, no spaces, in a directory
+        - project_euler/solution_test.py  # test project Euler solutions
```

Comment:
I believe that can be removed because pytest already discovers and runs this file.

Reply:
Ah, right, my bad. Do we want `pytest` to pickup this file as we want to fail fast? We can change the filename.

Reply:
> Do we want pytest to pickup this file

Yes.  We "get that for free" so it reduces configuration complexity.